scripts/auto-rebase/rebase_job_entrypoint.sh

```diff
@@ -18,14 +18,23 @@ release_amd64="$(oc get configmap/release-release-images-latest -o yaml \
 release_arm64="$(oc get configmap/release-release-images-arm64-latest -o yaml \
     | yq '.data."release-images-arm64-latest.yaml"' \
     | jq -r '.metadata.name')"
+# LVMS is not tracked in the OCP release image.  Instead, rely on the latest from the same X.Y stream as the release image.
+# Input:
+#   registry.ci.openshift.org/ocp/release:4.13.0-0.nightly-2023-01-17-152326
+# Output:
+#    "4.13"
+release_lvms=$( echo "$release_amd64" | sed -E 's,.*:(v?[0-9]+\.[0-9]+).*,\1,')
```

Comment:
How is this going to work in main branch? I mean, we have no 4.13 release of LVMS yet, don't we?